src/script/calling/CallingRepository.ts

```diff
@@ -692,7 +692,6 @@ export class CallingRepository {
     const convId = this.serializeQualifiedId(conversationId);
     this.logger.log(`Starting a call of type "${callType}" in conversation ID "${convId}"...`);
     try {
-      await this.checkConcurrentJoinedCall(conversationId, CALL_STATE.OUTGOING);
```

Comment:
It's not the responsibility of the CallingRepo to check if there is a call ongoing. This has been moved to the view layer (see `CallingViewModel`)